gutenberg.php

```diff
@@ -117,3 +117,11 @@ function gutenberg_pre_init() {
 
 	require_once dirname( __FILE__ ) . '/lib/load.php';
 }
+
+/**
+ * Outputs a WP REST API nonce.
+ */
+function gutenberg_rest_nonce() {
+	exit( wp_create_nonce( 'wp_rest' ) );
```

Comment:
Is this correct usage of `exit` , with the argument being interpreted as a status code ([documentation](https://www.php.net/manual/en/function.exit.php))? Is this intended primarily as shorthand to what could otherwise be expressed as...

```suggestion
	echo wp_create_nonce( 'wp_rest' );
	exit;
```

?

Reply:
A string is not interpreted as a status code. So yes, it's a shorthand of echo + exit. I think it's fine?